tests/integration/individual_response/test_individual_response.py

```diff
@@ -0,0 +1,375 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class IndividualResponseTestCase(IntegrationTestCase):
+    def setUp(self):
+        super().setUp()
+        self.launchSurvey("test_individual_response")
+
+    @property
+    def individual_section_link(self):
+        return self.getHtmlSoup().findAll(
+            "a", {"data-qa": "summary-actions-summary-button"}
+        )[1]["href"]
+
+    @property
+    def individual_response_link(self):
+        return (
+            self.getHtmlSoup()
+            .find("p", {"data-qa": "individual-response-url"})
+            .find_next()["href"]
+        )
+
+    @property
+    def previous_link(self):
+        return self.getHtmlSoup().find("a", {"data-ga-action": "Previous link click"})[
+            "href"
+        ]
+
+    @property
+    def sign_out_link(self):
+        return self.getHtmlSoup().find("a", {"name": "btn-save-sign-out"})["href"]
```

Comment:
I think it's worth adding this to `IntegrationTestCase` but making it work similar to `previous`.